docs/glossary.md

```diff
@@ -33,8 +33,11 @@ A sub-network established between peers via Transmission Control Protocol (TCP)
 #### bootstrapping
 During initial sync, the nano\_node requests old transactions to independently verify and populate its local ledger database. Bootstrapping will also occur when the nano\_node becomes out of sync with the network.
 
+#### burn
+When a 'burn' takes place, funds are sent to a specifc address that no one can access. Because no one can ever access funds sent to a burn address, it reduces the [curculating supply](#circulating-supply).
```

Comment:
Thanks for the updates, just a typo here with "curculating supply"

Reply:
Oops, thanks!